databases/week1/homework.sql

```diff
@@ -0,0 +1,53 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT(*)
+FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*)
```

Comment:
COUNT(*) as task_count
Pointer
Provide a column name for columns you have created